src/server/index.jsx

```diff
@@ -11,12 +11,19 @@ import path from 'path';
 // not part of react-helmet
 import helmet from 'helmet';
 import gnuTP from 'gnu-terry-pratchett';
+import dotenv from 'dotenv';
 import routes, { articleRegexPath } from '../app/routes';
 import { getStyleTag } from './styles';
 import getAssetsArray from './assets';
 
 import Document from '../app/components/Document';
 
+const result = dotenv.config();
```

Comment:
While we do need the env variables here, we also need them in webpack, can we also do this in the various webpack files?

Reply:
Good catch -- done.

Reply:
Should only need it in the `webpack.config.js` file - the other webpack config files are pulled into that one.